src/main/java/com/quorum/gauge/services/RawContractService.java

```diff
@@ -179,6 +190,78 @@
+        Quorum client = connectionFactory().getConnection(source);
+        Enclave enclave = buildEnclave(source, client);
+
+        String payload = base64SimpleStorageConstructorBytecode(initialValue);
+        SendResponse storeRawResponse = enclave.storeRawRequest(
+            payload,
+            privacyService.id(source),
+            Collections.emptyList()
+        );
+
+        String tmHash = base64ToHex(storeRawResponse.getKey());
+        return transactionService.sendSignedPrivateTransaction(apiMethod, tmHash, source, target, null);
+    }
+
+    public Observable<EthGetTransactionReceipt> updateRawSimplePrivateContractUsingEthApi(String apiMethod, int newValue, String contractAddress, QuorumNode source, QuorumNode target) {
+        Quorum client = connectionFactory().getConnection(source);
+        Enclave enclave = buildEnclave(source, client);
+
+        String payload = base64SimpleStorageSetBytecode(newValue);
+        SendResponse storeRawResponse = enclave.storeRawRequest(
+            payload,
+            privacyService.id(source),
+            Collections.emptyList()
+        );
+
+        String tmHash = base64ToHex(storeRawResponse.getKey());
+        EthSendTransaction sendTransactionResponse = transactionService.sendSignedPrivateTransaction(apiMethod, tmHash, source, target, contractAddress).toBlocking().first();
+
+        Optional<String> responseError = Optional.ofNullable(sendTransactionResponse.getError()).map(Response.Error::getMessage);
+        responseError.ifPresent(e -> logger.error("EthSendTransaction error: {}", e));
+
+        logger.debug("sent tx: {}", sendTransactionResponse.getTransactionHash());
+
+        return transactionService.getTransactionReceipt(source, sendTransactionResponse.getTransactionHash())
+            .map(ethGetTransactionReceipt -> {
+                if (ethGetTransactionReceipt.getTransactionReceipt().isPresent()) {
+                    return ethGetTransactionReceipt;
+                } else {
+                    throw new RuntimeException("retry");
+                }
+            }).retryWhen(new RetryWithDelay(20, 3000));
+    }
+
+    private String base64SimpleStorageConstructorBytecode(int initialValue) {
+        String hexBytecode = "0x6060604052341561000f57600080fd5b604051602080610149833981016040528080519060200190919050505b806000819055505b505b610104806100456000396000f30060606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680632a1afcd914605157806360fe47b11460775780636d4ce63c146097575b600080fd5b3415605b57600080fd5b606160bd565b6040518082815260200191505060405180910390f35b3415608157600080fd5b6095600480803590602001909190505060c3565b005b341560a157600080fd5b60a760ce565b6040518082815260200191505060405180910390f35b60005481565b806000819055505b50565b6000805490505b905600a165627a7a72305820d5851baab720bba574474de3d09dbeaabc674a15f4dd93b974908476542c23f00029";
```

Comment:
Let's read from `SimpleStorage.bin` instead of hardcode.
```java
InputStream binaryStream = SimpleStorage.class.getResourceAsStream("/com.quorum.gauge.sol/SimpleStorage.bin");
        if (binaryStream == null) {
            throw new IllegalStateException("Can't find resource ClientReceipt.bin");
        }
```